shell/platform/darwin/ios/framework/Source/FlutterAppDelegate_Test.h

```diff
@@ -7,9 +7,4 @@
 @interface FlutterAppDelegate (Test)
 @property(nonatomic, copy) FlutterViewController* (^rootFlutterViewControllerGetter)(void);
 
-- (BOOL)application:(UIApplication*)application
-            openURL:(NSURL*)url
-            options:(NSDictionary<UIApplicationOpenURLOptionsKey, id>*)options
-    infoPlistGetter:(NSDictionary* (^)())infoPlistGetter;
```

Comment:
I wound up getting rid of this test injection helper and mocking out `-[NSBundle objectForInfoDictionaryKey:]` instead.